web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
+- Navigating to the metrics page presents users with an empty query field and a prompt to enter a query in the box
+
+![Metrics page with too many results to graph](img/metrics-full.png)
+- If a query cannot be graphed, the results should still be shown in the table below, but the graph field will be populated by an error message until the issue is resolved or the offending query is switched off.
+
+![Metrics page with un-graphable results](img/metrics-error.png)
```

Comment:
Should the Job, Namespace, and Service entries include their resource badges and be a clickable blue color here, like the other screens?

Reply:
Updated